op-guide/ansible-deployment.md

```diff
@@ -209,6 +220,26 @@ location_labels = ["host"]
 
     - `capacity`: (DISK - log space) / TiKV instance number (the unit is GB)
 
+### Adjustment of inventory.ini variables
+
+| Variable | Description |
+| ---- | ------- |
+| cluster_name | the name of a cluster, adjustable |
+| tidb_version | the version of TiDB, configured by default in TiDB-Ansible branches |
```

Comment:
Does “branches” here stand for the master version or the 1.0 version?  If it is, please change branches to versions.

Reply:
The "branches" here means the actual branches of a GitHub repo, so just keep it.